Examples/Scripts/MaterialMaping/Mat_map_detector_plot_ratio.C

```diff
@@ -0,0 +1,105 @@
+// This file is part of the Acts project.
```

Comment:
We probably should converge on a naming convention for Root scripts, python/jupyter notebooks, etc.
@msmk0 what do you think?